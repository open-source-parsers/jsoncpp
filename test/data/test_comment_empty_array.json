[
  // Simple comment
  
  /* C-Style comment 
    second line
  */
]
