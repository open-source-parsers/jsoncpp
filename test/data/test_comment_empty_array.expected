.=[]
  // Simple comment
  
  /* C-Style comment 
    second linee
  */
